src/PresentationalComponents/StateView/StateView.test.js

```diff
@@ -16,11 +19,13 @@ describe('StatView', () => {
       </StateView>
     );
 
-    expect(toJson(wrapper)).toMatchSnapshot();
+    expect(queryByText(container, 'LOADING')).not.toBeNull();
+    expect(queryByText(container, 'DATA')).toBeNull();
```

Comment:
nit-pick:
```suggestion
    expect(queryByText('DATA')).not.toBeInTheDocument();
```

but this would also require importing '@testing-library/jest-dom'

Reply:
Done.